content/develop/use/patterns/indexes/index.md

```diff
@@ -19,16 +19,36 @@ Redis is not exactly a key-value store, since values can be complex data structu
 
 This document explains how it is possible to create indexes in Redis using the following data structures:
 
+* Hashes and JSON documents, using a variety of field types; used in conjunction with the Redis query engine.
 * Sorted sets to create secondary indexes by ID or other numerical fields.
 * Sorted sets with lexicographical ranges for creating more advanced secondary indexes, composite indexes and graph traversal indexes.
 * Sets for creating random indexes.
 * Lists for creating simple iterable indexes and last N items indexes.
+* Time series with labels.
 
 Implementing and maintaining indexes with Redis is an advanced topic, so most
 users that need to perform complex queries on data should understand if they
 are better served by a relational store. However often, especially in caching
 scenarios, there is the explicit need to store indexed data into Redis in order to speedup common queries which require some form of indexing in order to be executed.
 
+## Hashes and JSON indexes
+
+The Redis query engine provides capabilities to index and query both hash and JSON data using a variety of field types:
```

Comment:
Instead of "hash and JSON data" consider "hashes and JSON documents".

This index (+ time series) are different than the existing secondary indexes - instead of indexing data within a single key - here we index multiple keys, I think we should emphasize that.